src/transformers/configuration_t5.py

```diff
@@ -75,7 +75,6 @@ class T5Config(PretrainedConfig):
     def __init__(
         self,
         vocab_size=32128,
-        n_positions=512,
```

Comment:
T5 is not limited by a max number of positions since it uses relative pos embedding -> this param is never used in the Model File and should be deleted IMO: https://github.com/huggingface/transformers/issues/8047